plansys2_executor/test/unit/executor_test.cpp

```diff
@@ -224,13 +224,16 @@ TEST(executor, action_executor_client)
   auto test_node = rclcpp_lifecycle::LifecycleNode::make_shared("test_node");
   auto aux_node = rclcpp_lifecycle::LifecycleNode::make_shared("aux_node");
 
```

Comment:
As I said in the previous comment, let's maintain both constructors, so duplicate the test and modify the new test, and maintain this.